### PR TITLE
Clarified what get_node does.

### PR DESCRIPTION
**Pull request by RobertZenz**

### CODE REVIEW

**Comment by sofar** [comment]
yeah, that's a nice and detailed way to explain this. :+1:

**Comment by PilzAdam** [comment]
The format of nodes is already described earlier in the document.

File: doc/lua_api.txt [hunk 1]
```diff
@@ -1958,9 +1958,10 @@ and `minetest.auth_reload` call the authetification handler.
 * `minetest.remove_node(pos)`
     * Equivalent to `set_node(pos, "air")`
 * `minetest.get_node(pos)`
-    * Returns `{name="ignore", ...}` for unloaded area
+    * Returns the node at the given position as table in the format `{name="node_name", param1=0, param2=0}`,
```
Comment (nerzhul) [inline doc/lua_api.txt]: too long line

**Comment by RobertZenz** [comment]
I've rewrapped the lines.

**Comment by paramat** [comment]
:+1: Useful to have this information here.

**Comment by paramat** [comment]
386d190e0915a7ed270c313b4e9998f25d219e41

**Pull request closed without merging**